docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,306 @@
+This guide describes the process for making changes to the documentation whether you are an internal or external contributor.
 
-## Step 1/4 Set up your local environment
+## Before you begin
 
-Clone the `gravitational/docs` repository via `git clone`:
+Before you begin, verify that your environment meets the following prerequisites:
 
-```code
-$ git clone https://github.com/gravitational/docs
-```
+* You have a code editor, a GitHub account, and experience using command-line programs, including `git` commands and command line options.
 
-Different versions of the documentation are organized into the `content`
-directory of `gravitational/docs`. Each subdirectory of `content` includes a git
-submodule for a different branch of `gravitational/teleport`.
+* You have Node.js, version 14.x or newer, installed. 
+   Run `node --version` to check whether Node is installed.
+   If you need to install or update Node, see [Installing Node.js](https://nodejs.org/en/download/package-manager) for instructions to download and install Node.js using a package manager.
 
-Navigate to the root of your local clone of the `gravitational/docs` repository
-and run the following command to populate the `content` directory:
+* You have the `yarn` package manager installed.
+   Run `yarn --version` to check whether `yarn` is installed. 
+   If you need to install `yarn`, run the command `npm install --global yarn` to add it to your local system.
 
-```code
-$ yarn git-update
-```
+## Step 1/5. Set up your local environment
 
-Next, navigate to the directory under `content` that corresponds to the latest version of Teleport.
+To set up a local environment for contributing to Teleport documentation:
 
-```code
-$ cd content/(=teleport.major_version=).x
-```
+1. Open a terminal shell on your computer.
 
-Switch to the master branch (or versioned branch for updates specific to previous versions):
+1. Clone the `gravitational/docs` repository by running the following command:
 
-```code
-git switch master
-```
+    ```code
+    $ git clone https://github.com/gravitational/docs
+    ```
 
-Check out a new branch of `gravitational/teleport`.
+    This command clones the main branch for different versions of the documentation.
 
-```code
-$ git switch -c my-branch
-```
+1. Change to the root of the `docs` directory by running the following command:
+    
+    ```code
+    $ cd docs
+    ```
 
-## Step 2/4 Test your changes
+1. Initialize the submodules for documentation by running the following command:
+    
+    ```code
+    $ git submodule init
+    ```
 
-Run the following command to start the development server:
+1. Update the documentation to the latest version from the master branch by running the following command:
+    
+    ```code
+    $ yarn git-update
+    ```
 
-```code
-$ yarn install
-$ yarn dev
-```
+1. Install dependencies by running the following command:
```

Comment:
Maybe this should be the first step. If `git-update` ever depends on some node modules, it would have to come first.